kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -29,18 +23,18 @@ <h4 tabindex="-1">{{_('Support MDN, and it comes back to you.')}}</h4>
                 </p>
                 {% if is_popover %}
                     <div class="button-group">
-                        <a href="{{ faq_url }}" class="cta-link">
+                        <a href="{{url('payments')}}#contribute-faqs" class="cta-link">
```

Comment:
This change from PR #5052 should be kept

Reply:
Done